docs/streams/stream-basics/consumers.md

```diff
@@ -26,10 +26,14 @@ Cumulative acknowledgement cannot be used with [shared subscription mode](subscr
 
 ## Readers
 
-GDN stream readers are like stream consumers, but they have two crucial differences:
+Stream readers are similar to stream consumers, but there are two crucial differences between them:
 
-- You can specify where on a stream readers begin processing messages; consumers always begin with the latest available unacknowledged message.
-- Readers don't retain data or acknowledge messages.
+- Readers allow you to specify the starting point for processing messages within a stream. In contrast, consumers always begin with the latest available unacknowledged message.
```

Comment:
The consumers could begin from either the `earliest` or the `latest` but read only **unacknowledged** messages. 
There are 2 main differences between readers and consumers:
1. The readers don't acknowledge messages.
2. The readers reader both acknowledged and unacknowledged messages.